tests/conftest.py

```diff
@@ -47,3 +51,19 @@ def setup_shared_env():
         del os.environ["MINIO_ACCESS_KEY"]
         del os.environ["MINIO_SECRET_KEY"]
         del os.environ["MINIO_ENDPOINT"]
+
+
+@pytest.fixture
+def create_mock_secret():
+    def _get_secret(secret_name: str, secrets: dict[str, str]) -> list[Secret]:
+        keys = [SecretKeyValue(key=k, value=v) for k, v in secrets.items()]
+        new_secret = Secret(
+            owner_id=uuid4().hex,  # Assuming owner_id should be unique per secret
+            id=uuid4().hex,  # Generate a unique ID for each secret
+            name=secret_name,
+            type="custom",  # Assuming a fixed type; adjust as necessary
+        )
+        new_secret.keys = keys
```

Comment:
So this is the encryption step (because you're calling the [setter](https://github.com/TracecatHQ/tracecat/blob/4b2783028d47aca1b8ec159b1bc38bafebf19ae0/tracecat/db.py#L125-L130)), which why you're seeing a `bytes` object as your `encrypted_keys`